tests/cli-paths.rs

```diff
@@ -150,6 +150,21 @@ export PATH="$HOME/apple/bin"
         });
     }
 
+    #[test]
+    fn install_adds_path_to_rc_handling_no_newline() {
+        clitools::setup(Scenario::Empty, &|config| {
+            let profile = config.homedir.join(".profile");
+            let fake_rc_modified = FAKE_RC.strip_suffix('\n').expect("Should end in a newline");
+            raw::write_file(&profile, fake_rc_modified).unwrap();
+            expect_ok(config, &INIT_NONE);
+            expect_ok(config, &INIT_NONE);
```

Comment:
Why are you running rustup-init twice here?  As in, it's worth putting a coment in to explain what's going on, i.e. what we're trying to ensure does not rear its head once more.